ir-test.cxx

```diff
@@ -218,38 +215,102 @@ namespace ir {
 		}
 	};
 
-	void find_tests_in_dir(std::string& test_dir, std::vector<std::string>& irt_files) {
-		for (const std::filesystem::directory_entry& ent :
-		        std::filesystem::recursive_directory_iterator(test_dir)) {
-			std::string fl(ent.path().string());
-			if (fl.length() < 4 || 0 != fl.compare(fl.length()-4, 4, ".irt")) continue;
-			irt_files.push_back(fl);
+	void find_tests_in_dir(std::set<std::string>& user_files, std::vector<std::string>& irt_files) {
+		assert(user_files.size()>0);
+
+		for (const auto& f : user_files) {
+			if (std::filesystem::is_directory(f)) {
+				for (const std::filesystem::directory_entry& ent :
+		             std::filesystem::recursive_directory_iterator(f)) {
+					std::string fl(ent.path().string());
+					if (fl.length() < 4 || ent.path().extension() != ".irt")
+						continue;
+					irt_files.push_back(fl);
+                }
```

Comment:
nit: Looks like WS issue, closing brace with wrong indent. Thanks

Reply:
I should have checked diff in the website. Let me check what happened in my editor.